runelite-client/src/main/java/net/runelite/client/plugins/inferno/InfernoPlugin.java

```diff
@@ -1,439 +1,469 @@
+ */
+package net.runelite.client.plugins.inferno;
+
+import com.google.inject.Provides;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Actor;
+import net.runelite.api.ChatMessageType;
+import net.runelite.api.Client;
+import net.runelite.api.GameState;
+import net.runelite.api.HeadIcon;
+import net.runelite.api.NPC;
+import net.runelite.api.NpcID;
+import net.runelite.api.events.AnimationChanged;
+import net.runelite.api.events.ChatMessage;
+import net.runelite.api.events.ConfigChanged;
+import net.runelite.api.events.GameStateChanged;
+import net.runelite.api.events.GameTick;
+import net.runelite.api.events.NpcDespawned;
+import net.runelite.api.events.NpcSpawned;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.plugins.PluginType;
+import net.runelite.client.ui.overlay.OverlayManager;
+import org.apache.commons.lang3.ArrayUtils;
+
+@PluginDescriptor(
+		name = "Inferno",
+		description = "Inferno helper",
+		tags = {"combat", "overlay", "pve", "pvm"},
+		type = PluginType.PVM
+)
+@Slf4j
+public class InfernoPlugin extends Plugin
+{
+	private static final int INFERNO_REGION = 9043;
+
+	@Inject
+	private Client client;
+
+	@Inject
+	private OverlayManager overlayManager;
+
+	@Inject
+	private InfernoOverlay infernoOverlay;
+
+	@Inject
+	private InfernoWaveOverlay waveOverlay;
+
+	@Inject
+	private InfernoJadOverlay jadOverlay;
+
+	@Inject
+	private InfernoInfobox infernoInfobox;
+
+	@Inject
+	private InfernoNibblerOverlay nibblerOverlay;
+
+	@Inject
+	private InfernoConfig config;
+
+	@Getter
+	private int currentWave = -1;
+
+	@Getter
+	private Map<NPC, InfernoNPC> monsters;
+
+	@Getter
+	private Map<Integer, ArrayList<InfernoNPC>> monsterCurrentAttackMap;
+
+	@Getter
+	private List<NPC> nibblers;
+
+	@Getter
+	private InfernoNPC[] priorityNPC;
+
+
+	@Getter(AccessLevel.PACKAGE)
+	@Nullable
+	private InfernoJadAttack attack;
+
+	private NPC jad;
+
+	@Getter(AccessLevel.PACKAGE)
+	private int currentWaveNumber;
+
+	@Getter(AccessLevel.PACKAGE)
+	private int nextWaveNumber;
+
+	private List<Actor> waveMonsters;
+
+	public InfernoPlugin()
+		{
+			waveMonsters = new ArrayList<>();
+		}
+
+	@Provides
+	InfernoConfig provideConfig(ConfigManager configManager)
+		{
+			return configManager.getConfig(InfernoConfig.class);
+		}
+
+	@Override
+	protected void startUp() throws Exception
+	{
+		if (isInInferno())
+		{
+			overlayManager.add(infernoOverlay);
+			overlayManager.add(infernoInfobox);
+			overlayManager.add(nibblerOverlay);
+
+			if (config.waveDisplay() != InfernoWaveDisplayMode.NONE)
+			{
+				overlayManager.add(waveOverlay);
+			}
```

Comment:
delete 
			if (config.waveDisplay() != InfernoWaveDisplayMode.NONE)
			{
				overlayManager.add(waveOverlay);
			}

add overlayManager.add(waveOverlay); to startup

Reply:
No, wanna go away from having overlays statically on so their render methods don't get called